src/shrutiS/assignment22/PrimeNumbers.java

```diff
@@ -0,0 +1,25 @@
+/*Assignment-22 : 18th Nov'2020 
+Print all prime numbers between 2 to 100.*/
+package shrutiS.assignment22;
+
+public class PrimeNumbers {
+
+	void findPrimeNumbers() {
+		for (int index = 2; index <= 100; index++) {
+			int count = 0;
+			for (int innerIndex = 1; innerIndex <= index / 2; innerIndex++) {
```

Comment:
Hi @ShrutiSOct2020 
We cannot consider as prime or non-prime so ideally loop should start from 2 onwards.